packages/compartment-mapper/src/parse-archive-cjs.js

```diff
@@ -1,13 +1,13 @@
 // @ts-check
-
-/** @typedef {import('ses').ThirdPartyStaticModuleInterface} ThirdPartyStaticModuleInterface */
+/// <reference types="ses"/>
```

Comment:
Interesting. I'd love if we could switch the base `ses` away from ambient types.

Reply:
Base SES provides certain ambient types like `Compartment`, `harden`, and `lockdown`, but ThirdPartyStaticModuleInterface is not ambient. I was able to remove this line.